addons/web/static/src/views/form/form_controller.scss

```diff
@@ -416,6 +416,11 @@
     .o_group {
         .o_form_label {
             font-weight: $font-weight-normal;
+
+            & .o_sup_tooltip {
+                cursor: default;
```

Comment:
You may have used `cursor-default` utility class.
Btw, not needed anyways since  we'll remove the `btn`ones.

Reply:
Thanks! I take note for future use case